apps/bletiny/src/gatt_svr.c

```diff
@@ -366,7 +366,7 @@ static int
 gatt_svr_access_test(uint16_t conn_handle, uint16_t attr_handle,
                      struct ble_gatt_access_ctxt *ctxt,
                      void *arg)
-{
+
```

Comment:
I agree, it was indeed a complicated merge. Thank you for the review  @utzig and @devlware. I fixed it in a subsequent commit.